src/components/ui/Header.jsx

```diff
@@ -0,0 +1,131 @@
+"use client"
+
+import { useState } from 'react';
+import { Dialog, Popover } from '@headlessui/react';
+import { Bars3Icon, XMarkIcon } from '@heroicons/react/24/outline';
+import { Globe } from 'lucide-react';
+import { Button } from "@/components/ui/Button";
+import Logo from '../../images/mjl-logo_sq.png';
+import Link from 'next/link';
+
+function Header() {
+  const [mobileMenuOpen, setMobileMenuOpen] = useState(false);
+
+  return (
+    <header className="bg-white">
+      <nav className="mx-auto flex max-w-7xl items-center justify-between p-6 lg:px-8" aria-label="Global">
+        <div className="flex lg:flex-1">
+          <Link href="/" className="-m-1.5 p-1.5">
+            <span className="sr-only">Your Company</span>
+            <img className="h-8 w-auto" src={Logo.src} alt="logo" />
```

Comment:
reiketu naudoti  Image taga is nexto https://nextjs.org/docs/app/building-your-application/optimizing/images

Reply:
Fixed, new commit